mne/beamformer/_dics.py

```diff
@@ -140,14 +309,17 @@ def dics(evoked, forward, noise_csd, data_csd, reg=0.05, label=None,
     real_filter : bool
         If True, take only the real part of the cross-spectral-density matrices
         to compute real filters as in [2]_. Default is False.
+    picks : array-like of int
+        Channel indices to use for beamforming (if None all channels
+        are used except bad channels).
```

Comment:
Mh, I don't know about design principles here :wink:, but wouldn't it be a handy option? Otherwise you'd have to adjust ``data_cov``, ``noise_cov`` and ``forward`` manually if you wanted to check out a channel selection?